packages/wdio-cli/tests/watcher.test.ts

```diff
@@ -29,20 +37,31 @@ jest.mock('../src/launcher', () => {
     return LauncherMock
 })
 
-class WorkerMock {
+interface WorkerMockRunPayload extends Partial<Omit<Capabilities.WorkerRunPayload, 'cid'>> {
+    cid?: string | null;
+    isBusy?: boolean;
+    sessionId?: string | null;
```

Comment:
Do we actually assign `null` to them at some point?
```suggestion
    cid?: string
    isBusy?: boolean
    sessionId?: string
```

Reply:
Good catch, thank you — this might have been left over from a time before I refactored the `WorkerMock` constructor. Addressed in 1b95b1e.